packages/form/src/components/Action.vue

```diff
@@ -36,11 +36,43 @@
     </div>
     <div class="col-12">
       <div class="form-group" id="group.action.attachments">
-        <label :for="`action.attachments`"> {{ $t(`Attachement (s)`) }} </label>
+        <label :for="`action.attachments`"> {{ $t(`Attachment (s)`) }} </label>
         <Links  @input="update" id="action.attachments" v-model="action.attachments" :type="[ 'files','links' ]" multi name="attachments"/>
       </div>
     </div>
-    
+
+    <div class="col-lg-3">
+      <div class="form-group" id="group.action.startDate">
+        <label  for="action.startDate"> {{ $t(`Start Date`) }} </label>
+        <input class="form-control" 
+          @input      ="update"
+          id          ="action.startDate"
+          type        ="date"
+          v-model.trim="action.startDate[$i18n.locale]"
+          v-validate  ="'required'"
+          :class      ="[ getValidationClass(`action.startDate`) ]"
+          :name       ="`action.startDate`"
+          />
+        <field-error-message :error="errors.collect('action.startDate')"/>
+      </div>
+    </div>
+
+    <div class="col-lg-3">
+      <div class="form-group" id="group.action.enddate">
+        <label  for="action.endDate"> {{ $t(`End Date`) }} </label>
+        <input class="form-control" 
+          @input      ="update"
+          id          ="action.endDate"
+          type        ="date"
+          v-model.trim="action.endDate[$i18n.locale]"
+          v-validate  ="'required'"
```

Comment:
not required, need further validation.  Can the end be before the start?  
https://vee-validate.logaretm.com/v2/guide/rules.html#date-between